deploy/Chart/templates/controller/rbac.yaml

```diff
@@ -52,6 +56,20 @@ rules:
   - '*'
   verbs:
   - '*'
+- apiGroups:
+  - source.toolkit.fluxcd.io
+  resources:
+  - gitrepositories
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - source.toolkit.fluxcd.io
+  resources:
+  - gitrepositories/status
+  verbs:
+  - get
```

Comment:
Is this part of this PR or something we need for the next one?

Reply:
part of the next one. I removed it for now